packages/tangle/tipmanager.go

```diff
@@ -375,7 +375,7 @@ func (t *TipManager) isPastConeTimestampCorrect(messageID MessageID) (timestampV
 	}
 
 	// if last confirmed message if older than minSupportedTimestamp, then all tips are invalid
-	if t.tangle.TimeManager.LastConfirmedMessage().Time.Before(minSupportedTimestamp) {
+	if t.tangle.TimeManager.CurrentTangleTime().Before(minSupportedTimestamp) {
```

Comment:
We do not want to invalidate tips if we stop confirmations. I think this time should be RCTT.

Reply:
Done

Reply:
I am sorry, but I was actually wrong; here we have to evaluate the difference between CTT (which won't advance if confirmations stop) and the timestamp of the oldest confirmed message of the tip's past cone. If we would compare the latter with RCTT or wall clock we will end up invalidating all tips.

Reply:
This is updated in another PR, so we can resolve it there and not change it here.

Reply:
@karimodm do you agree with @piotrm50? Can I resolve and leave it as is?

Reply:
Yeah, I think we should merge this PR into the other.